numba_dpex/dpnp_iface/dpnp_ufunc_db.py

```diff
@@ -72,20 +74,38 @@ def _fill_ufunc_db_with_dpnp_ufuncs(ufunc_db):
                 "d->d": mathimpl.lower_ocl_impl[("erf", (_unary_d_d))],
             }
         else:
-            op = getattr(dpnp, ufuncop)
+            dpnpop = getattr(dpnp, ufuncop)
             npop = getattr(np, ufuncop)
-            op.nin = npop.nin
-            op.nout = npop.nout
-            op.nargs = npop.nargs
-            op.types = npop.types
-            op.is_dpnp_ufunc = True
+            if not hasattr(dpnpop, "nin"):
+                dpnpop.nin = npop.nin
+            if not hasattr(dpnpop, "nout"):
+                dpnpop.nout = npop.nout
+            dpnpop.nargs = dpnpop.nin + dpnpop.nout
```

Comment:
Let us add a `if not hasattr(dpnpop, "nargs")` guard here too to prevent future error when dpnp adds `nargs` attribute.